### PR TITLE
fix: solved issue 6585

### PR DESCRIPTION
**Pull request by Cold-Martijn**
https://github.com/azerothcore/azerothcore-wotlk/issues/6585

## Changes Proposed:
-  Set wanderdistance for specific GUIDs
-  Set movementtype for them aswell.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6585

## Tests Performed:

- checked ingame
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1631616272860535120.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+UPDATE creature SET wander_distance = 5 WHERE guid = 56058;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56059;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56086;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56127;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56160;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56162;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56125;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56123;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56166;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56150;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1631616272860535120.sql:12]: Add backticks and make this one query please https://www.azerothcore.org/wiki/sql-standards#compact-queries

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1631616272860535120.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1631616272860535120.sql:12]: Also when working in the creature table it is preferred if you update guid AND id

**Review by Kitzunu — APPROVED** [review]

**Comment by Cold-Martijn** [comment]
Fixed an extra ; between GUID (..., , .... , ...); AND ID, sorry :)

**Review by Kitzunu — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@Cold-Martijn welcome to AzerothCore and thanks for your contribution!

@Kitzunu don't forget to apply the workflow labels ;)

**Pull request merged**